asset/asset.go

```diff
@@ -1958,6 +1958,18 @@ func (a *Asset) Leaf() (*mssmt.LeafNode, error) {
 	return mssmt.NewLeafNode(buf.Bytes(), a.Amount), nil
 }
 
+// Specifier returns the asset's specifier.
+func (a *Asset) Specifier() Specifier {
+	id := a.Genesis.ID()
+	specifier := NewSpecifierFromId(id)
+
+	if a.GroupKey != nil {
```

Comment:
See previous comment, could just be `return NewSpecifierWithGroup(a.genesis.ID(), a.GroupKey)`.